pkg/services/login/authinfoservice/database/stats.go

```diff
@@ -60,7 +60,26 @@ func (s *AuthInfoStore) RunMetricsCollection(ctx context.Context) error {
 	}
 }
 
-func (s *AuthInfoStore) GetLoginStats(ctx context.Context) (login.LoginStats, error) {
+func (s *AuthInfoStore) CollectLoginStats(ctx context.Context) (map[string]any, error) {
```

Comment:
@eleijonmarck @Jguer Do you think we can remove this one? I remember it had performance issues so we have not used it for quite some time

Reply:
I think it's ok to remove it :+1: